changelog.json

```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.2.4-beta4": [
+      "[Fixed] Compare tab appends older commits when scrolling to bottom of list - #4964",
```

Comment:
Should we call out the new notification banner or hold off until next week?

Reply:
That seems dependent on merging #4993 and shipping another beta. I think it can wait.